packages/api/src/utils.js

```diff
@@ -59,26 +86,54 @@ const decodeStateTransition = async (client, base64) => {
 
       decoded.identityId = stateTransition.getIdentityId().toString()
       decoded.amount = output.satoshis * 1000
+      decoded.signature = stateTransition.getSignature()?.toString('hex') ?? null
+      decoded.raw = stateTransition.toBuffer().toString('hex')
```

Comment:
Missing assetlockproof and funding address (input) data

Reply:
1/2 ✅